android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -985,4 +986,25 @@ class HmssdkFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware,
         getLocalPeer().videoTrack?.setMute(!(allowed!!))
         result?.success(null)
     }
+
+    private fun setVolume(call: MethodCall){
+        val trackId = call.argument<String>("track_id")
+        val volume = call.argument<Double>("volume")
+        val peer=hmssdk.getPeers().first {
+            it.audioTrack?.trackId == trackId
```

Comment:
check-in the auxiliary tracks of the peer as well. 
If the peer has auxiliary track, & if it's of RemoteAudioTrack type & trackId matches then set volume